perftest/simulations/README.md

```diff
@@ -27,7 +27,7 @@ class SimpleSimulation extends Simulation {
     .client(
       NessieClient
         .builder()
-        .withUri("http://127.0.0.1:19120/api/v1")
+        .withUri(s"http://127.0.0.1:${System.getProperties.getProperty("quarkus.http.test-port")}/api/v1")
```

Comment:
could it make sense to put this into https://github.com/projectnessie/nessie/blob/a66390b1e4f2ba73e21c067b3576ec20dffb22e0/perftest/simulations/src/test/scala/org/projectnessie/perftest/gatling/BaseParams.scala#L57-L60 ?

Reply:
I'd be fine without refactoring the client creation into a common place in this PR, but if it comes to that, please see my suggestion of `ProtocolParams` in #3199

Reply:
yeah, I think we can make it common in a follow-up PR